eth/downloader/downloader.go

```diff
@@ -1112,9 +1115,12 @@ func (d *Downloader) fillHeaderSkeleton(from uint64, skeleton []*types.Header) (
 	d.queue.ScheduleSkeleton(from, skeleton)
 
 	err := d.concurrentFetch((*headerQueue)(d))
-	log.Debug("Skeleton fill terminated", "err", err)
-
+	if err != nil {
+		log.Debug("Skeleton fill failed", "err", err)
+	}
 	filled, proced := d.queue.RetrieveHeaders()
+	log.Debug("Skeleton fill succeeded", "filled", len(filled), "processed", proced)
```

Comment:
This would print both failed and succeeded

Reply:
Ugh, just added that while debugging :)) Lemme fix it.